src/system-config.ts

```diff
@@ -7,12 +7,27 @@
  **********************************************************************************************/
 /** Map relative paths to URLs. */
 const map: any = {
+    '@angular2-material': 'vendor/@angular2-material'
 };
 
 /** User packages configuration. */
 const packages: any = {
 };
 
+
+// put the names of any of your Material components here
+const materialPkgs:string[] = [
```

Comment:
Do we need to this by our hands? There is a lot for a single task like adding material design.